client/src/components/Ratings/sub-components/ReviewTile.jsx

```diff
@@ -27,12 +27,33 @@ export default function ReviewTile({
     const monthName = months[date.getMonth()];
     const day = date.getDate();
     const year = date.getFullYear();
-
     return `${monthName} ${day}, ${year}`;
   };
 
+  // Creates name and Date string for each tile.
   const nameAndDate = `${getDateString(review.date)}, ${review.reviewer_name}`;
 
+  // Highlights text that matches the searchTerm, set by ReviewSearchBar.jsx
+  const highlightHandler = (text, fn) => {
+    if (searchTerm.length < 3) return fn ? fn() : text;
```

Comment:
return `fn()` if `fn` is falsy? not sure I understand. could you comment on this or get back with me?